src/components/PersonLink.tsx

```diff
@@ -0,0 +1,56 @@
+import { FC } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../types';
+
+type Props = {
+  person: Person;
+  checkingPersonName: (motherName: string | null) => Person | null;
+};
+
+export const PersonLink: FC<Props> = ({ person, checkingPersonName }) => {
+  const hasMotherInList = checkingPersonName(person.motherName);
```

Comment:
it's better to use destructuring assignment to get all properties from the `person` you need